ngraph/frontend/onnx_import/src/op/constant_fill.cpp

```diff
@@ -0,0 +1,67 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <onnx/onnx_pb.h> // onnx types
+
+#include "default_opset.hpp"
+#include "exceptions.hpp"
+#include "ngraph/op/broadcast.hpp"
+#include "ngraph/op/concat.hpp"
+#include "ngraph/op/constant.hpp"
+#include "onnx_common/utils.hpp"
+
+namespace ngraph
+{
+    namespace onnx_import
+    {
+        namespace op
+        {
+            namespace set_1
+            {
+                // ConstantFill is a deprecated experimental operator removed in ONNX 1.4
+                OutputVector constant_fill(const Node& node)
+                {
+                    Output<ngraph::Node> target_shape;
+                    const auto fill_value = node.get_attribute_value<float>("value", 0.f);
+                    const auto dtype = node.get_attribute_value<int64_t>(
+                        "dtype", static_cast<int64_t>(ONNX_NAMESPACE::TensorProto_DataType_FLOAT));
+                    const auto ng_type = onnx_common::onnx_to_ng_data_type(
+                        static_cast<ONNX_NAMESPACE::TensorProto_DataType>(dtype));
+                    const auto const_val_to_fill = std::make_shared<default_opset::Convert>(
+                        default_opset::Constant::create(element::f32, {}, {fill_value}), ng_type);
```

Comment:
Why did you use convert instead of create a constant with the right type?
```suggestion
                    const auto const_val_to_fill = default_opset::Constant::create(ng_type, {}, {fill_value});
```

Reply:
It seems to be an artifact from some test version ;)